chrome-extension/open-ai-fetch-interceptor/fetch_interceptor.js

```
@@ -0,0 +1,242 @@
+                  jsonObject.id = jsonPart.v.message.id;
+                  jsonObject.children = [];
+                }
+
+                // Handle content accumulation
+                if (typeof jsonPart.v === 'string') {
+                  fullResponseText += jsonPart.v;
+                }
+                else if (jsonPart.p && jsonPart.p.startsWith('/message/content/parts/') && typeof jsonPart.v === 'string') {
+                  fullResponseText += jsonPart.v;
+                }
+                else if (Array.isArray(jsonPart.v)) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)